add-chain/main.go

```diff
@@ -204,11 +204,14 @@ func getGasPayingToken(l1rpcURl string, SystemConfigAddress superchain.Address)
 
 	opts := bind.CallOpts{}
 	result, err := sc.GasPayingToken(&opts)
+
 	if strings.Contains(err.Error(), "execution reverted") {
 		// This happens when the SystemConfig contract
 		// does not yet have the CGT functionality.
 		return nil, nil
-	} else if err != nil {
+	}
+
+	if err != nil {
```

Comment:
I think this was actually fine before.

Reply:
Yeah its a minor one / style / personal taste. In a more complicated function, I like doing this because it can make it easier to read https://danp.net/posts/reducing-go-nesting/ .